board/app/controllers/thread_controller.php

```diff
@@ -0,0 +1,108 @@
+<?php
+class ThreadController extends AppController {
+    
+/*Create a new comment (Thread)
+* :: - STATIC FUNCTION, can be called from the class name
+* -> - INSTANCE, can only be called from an instance of the class.
+*/
+public function create(){
```

Comment:
Please indent all codes inside this class :bow:  (and apply to others~)
Then indent all included under the functions as well~